---
title: "Ongoing Research"
layout: single
author_profile: true
permalink: /ongoing-research/
excerpt: "<br /> <br /> <br /> <br /> <br /> <br /> <br />"
header:
  overlay_image: austinhpatton.github.io/Images/PjorOverhead.jpg
  overlay_filter: 0.3
  caption: "<i>Plethodon jordani</i>, Dan Mele Photography"
---
<h2>Assessing the role of hybridization in macroevolutionary diversification</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/Fig1_ModAvg_RateState_Spec.jpg">
</figure>
<font size="3"> Whether hybridization generates or erodes species diversity has long been debated, but to date most studies have been conducted at small taxonomic scales. Salamanders (order Caudata) represent a taxonomic order in which hybridization plays a prevalent ecological and evolutionary role. In this study, we employ a recently developed model of trait-dependent diversification, HiSSE, to test the hypothesis that speciation and extinction rates will differ between hybridizing and non-hybridizing taxa. We find strong evidence supporting this hypothesis, showing that hybridizing salamander lineages have significantly greater net-diversification rates than non-hybridizing lineages. This pattern is driven by concurrently increased speciation rates and decreased extinction rates in hybridizing lineages. Our results support the hypothesis that hybridization can act as a generative force in macroevolutionary diversification. A video of my recent presentation at Evolution 2017 on this research is posted below. 
This work is in collaboration with <u><a href="http://public.wsu.edu/~storfer/">Andrew Storfer (PI)</a></u>, <u><a href="http://lukejharmon.github.io/">Luke Harmon</a></u>, Mark Margres, Brendan Epstein, and <u><a href="https://public.wsu.edu/~storfer/eastman/pages/index.html">Jon Eastman</a></u>. <iframe width="560" height="315" src="https://www.youtube.com/embed/Tkuhtg3x0AA?list=PLWCsWxY07Dh5dIhFwrTtLcJBZK81Wp1Xn" frameborder="0" allowfullscreen></iframe></font>


<h2>Colonization mediated priority effects drive explosive radiation of mainland Anolis lizards</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/Figure1_HiSSE-BAMM-DiversificationRates.jpg" class="inline">
</figure>
<font size="3">Adaptive radiation has long captured the interest of evolutionary biologists. Defined as the accumulation of ecologically differentiated, coexisting species within a rapidly diversifying lineage, adaptive radiations have long been treated as model systems to study the intersection of ecology and evolution. Despite decades of study however, there exists little consensus regarding what governs the tempo and mode of adaptive radiation. Often occurring following colonization of a new, geographically isolated and ecologically distinct landscape, adaptive radiations have most frequently been described and studied in island-like systems. The consequence of this tendency has led to a paradigm in which adaptive radiations are almost synonymized with islands and lakes. In this study, we show that colonization of upper Central America by Anolis lizards led to their rapid radiation on the mainland. Importantly, timing of colonization plays a crucial role in the eventual outcome of diversification; Central America and the Nearctic was at the time isolated from South America and harbored no Anoline diversity, thus enabling this radiation. We find that although colonization can lead to dramatic shifts in diversification dynamics, patterns of morphological rate variation are decoupled from lineage diversification rates, yet predict morphological disparity. Further, we present evidence of widespread convergence in mainland anoles, exceeding that observed within island lineages. Our study emphasizes the need to focus research efforts on the study of historically understudied mainland diversity. Although radiations on continental landmasses are typically less readily circumscribed, our study demonstrates that these radiation may harbor equally, if not more exceptional diversity than their island counterparts. 
This work is in collaboration with <u><a href="http://lukejharmon.github.io/">Luke Harmon</a></u>, <u><a href="https://mahlerlab.com/">Luke Mahler</a></u>, <u><a href="http://www.anthonyherrel.fr/">Anthony Herrel</a></u>, and <u><a href="https://sites.wustl.edu/losos/">Jonathan Losos</a></u>.</font>

<h2>Comparative transcriptomics of hybridization</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/CompLangGenomics_Image.jpg" class="inline">
</figure>
<font size="3">The Southern Appalachians contain the greatest diversity of salamanders in the world. Of the salamanders native to this region, the <i>Plethodontid</i> (terrestrial lungless) salamanders are the most diverse, and many of these have been observed to hybridize in nature. In some cases, hybrizing species pairs hybridize in some locations but not in others. This appreciable frequency of hybridization thus lends to the question of what the consequences of hybridization are for these species. One means by which we may address this is through the use of transcriptomics, thus characterizing patterns of gene expression across parental taxa, relative to hybrids. We have assembled and are currently annotating the (tail) transcriptomes of three species of eastern plethodontid salamanders (<i>Plethodon jordani, Plethodon metcalfi, Plethodon teyahalee</i>) that hybridize frequently in nature. In an upcoming series of projects, we aim to leverage these transcriptomes and those of their hybrids to identify genomic regions associated with differential expression, both among parental taxa and aon hybrids. By relating these differentially expressed genes back to their functional annotations, we thus may gain insight into 1) what genes lead to interspecific differences, and 2) aspects of the transcriptome most frequently impacted by hybridization among taxa. 
This work is in collaboration with <u><a href="http://public.wsu.edu/~storfer/">Andrew Storfer (PI)</a></u>, <u><a href="https://kelleylab.wordpress.com/">Joanna Kelley</a></u> and <u><a href="https://scholar.google.com/citations?user=HyuvK6AAAAAJ&hl=en">Mark J. Margres</a></u>.</font>

<h2>Comparison of contemporary demographic reconstruction methods</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/SMC_Sim_Compare.png" class="inline">
</figure>
<font size="3">Reconstructing species' demographic histories is a central focus of molecular ecology and evolution. Recently, an expanding suite of methods leveraging either the sequentially Markovian coalescent (SMC) or the site-frequency spectrum (SFS) have been developed to reconstruct population size histories from genomic sequence data. However, few studies have investigated the robustness of these methods to genome assemblies of varying quality. In this study, we presented an improved genome assembly for the Tasmanian devil using the Chicago library method. Compared to the original reference genome, our new assembly reduced the number of scaffolds (from 35,975 to 10,010) and increased the scaffold N90 (from 0.101 to 2.164 Mb). Second, we assessed the performance of four contemporary genomic methods for inferring population size history (PSMC, MSMC, SMC++, Stairway Plot), using the two devil genome assemblies as well as simulated, artificially fragmented genomes that approximate the hypothesized demographic history of Tasmanian devils. We demonstrated that each method is robust to assembly quality, producing similar estimates of N<sub>e</sub> up to around 5,000 fragments. Overall, methods reliant on the SMC are most reliable between ~300 generations before present (gbp) and 100 kgbp, whereas methods exclusively reliant on the SFS are most reliable between the present and 30 gbp. Our results suggest that whole-genome methods for reconstructing species' effective population size histories: 1) can be applied to non-model organisms without highly contiguous reference genomes; 2) are capable of detecting independently documented effects of historical geological events; and, 3) should be used in concert.</font>

<h2>Phylodynamics of a transmissible cancer</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/DFTD_Pic.jpg" class="inline">
</figure>
<font size="3">Of paramount importance to the conservation of species diversity is an understanding of emerging infectious diseases (EIDs). Frequently driven by novel interactions among previously isolated species, EIDs have been implicated in recent declines or even extinctions in the cases of Dutch elm disease, chytrid fungus in amphibians, and avian malaria an Hawaiian birds. Given the increased prevalence of habitat alteration and severity of global climate change, geographic ranges of many species, including vectors of EIDs are expected to change dramatically. Such range shifts have the potential to facilitate both the spread and establishment of EIDs, as was observed recently with the emergence of Zika virus in Brazil of early 2015. In this study, we aim to characterize the transmission dynamics among lineages of one such EID, the Tasmanian Devil Facial Tumor Disease (DFTD), using a phylodynamic approach. Leveraging low-coverage whole-genome sequencing of 50 serially-sampled tumors collected from Tasmanian Devils <i>Sarcophilus harrisii</i>, we will be able to identify tumor strains exhibiting greater virulance. With this information, we then may begin to identify those components of DFTD evolution that led to these increased lineage fitnesses. 
This work is in collaboration with <u><a href="http://public.wsu.edu/~storfer/">Andrew Storfer (PI)</a></u>, <u><a href="https://www.mccallum-disease-ecology.com/">Hamish McCallum (Co-PI)</a></u>, <u><a href="http://www.utas.edu.au/zoology/research/conservation-biology-and-wildlife-management/effects-of-dftd-on-behaviours-associated-with-transmission-and-on-reproductive-females/menna-jones">Menna Jones (Co-PI)</a></u>, <u><a href="http://hohenlohelab.github.io/">Paul Hohenlohe</a></u>, and <u><a href="https://scholar.google.com/citations?user=HyuvK6AAAAAJ&hl=en">Mark Margres</a></u>.</font>



