---
title: "Ongoing Research"
layout: single
author_profile: true
permalink: /ongoing-research/
excerpt: "<br /> <br /> <br /> <br /> <br /> <br /> <br />"
header:
  overlay_image: austinhpatton.github.io/Images/PjorOverhead.jpg
  overlay_filter: 0.3
  caption: "<i>Plethodon jordani</i>, Dan Mele Photography"
---
<h2>Assessing the role of hybridization in macroevolutionary diversification</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/HybDivers_Image.jpg">
</figure>
<font size="3"> The role of hybridization has had a long contentious role in biology. Thought of as either facilitating and accelerating the lineage diversification process or as constraining it, the majority of research conducted on this topic has been at relatively small taxonomic scales. To address this question in extant salamanders, we leveraged a recently developed model of trait-dependent diversification (Hidden-State Speciation and Extinction: HiSSE). We have found that hybridizing species experience nearly 6-times greater speciation rates than non-hybridizing taxa, resulting in heightened net-diversification rates. Thus, our results favor a view of hybridization as a generative process in species diversification. This interpretation is important, both for our understanding of macroevolutionary diversification, and for our understanding of contemporary hybridization among native species. Given the increased frequency of hybridization among previously isolated species due to habitat alteration and climate-change mediated range shifts, such an understanding of hybridization in the diversification process will contribute meaningfully to discussions of conservation management. This work is in collaboration with <u><a href="http://public.wsu.edu/~storfer/">Andrew Storfer (PI)</a></u>, <u><a href="http://lukejharmon.github.io/">Luke Harmon</a></u>, Brendan Epstein, and <u><a href="https://public.wsu.edu/~storfer/eastman/pages/index.html">Jon Eastman</a></u>.</font>

<h2>Investigating the population and landscape genomics of hybridization</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/CompLangGenomics_Image.jpg" class="inline">
</figure>
<font size="3">The Southern Appalachians contain the greatest diversity of salamanders in the world. Of the salamanders native to this region, the <i>Plethodontid</i> (terrestrial lungless) salamanders are the most diverse, and many of these have been observed to hybridize in nature. In some cases, hybrizing species pairs hybridize in some locations but not in others. This incomplete reproductive isolation begs the question of how this group came to become so diverse if hybridization is so frequent. In an upcoming series of projects, we aim to leverage population and landscape genomic approaches to address this question. Specifically, we aim to identify genomic regions associated with local adaptation and adaptive introgression among three hybridizing species (<i>Plethodon jordani, Plethodon metcalfi, Plethodon teyahalee</i>). To accomplish this, we will use sequence capture to target loci found to be differentially expressed using transcriptomic data. This work is in collaboration with <u><a href="http://public.wsu.edu/~storfer/">Andrew Storfer (PI)</a></u>, <u><a href="https://kelleylab.wordpress.com/">Joanna Kelley</a></u> and <u><a href="https://scholar.google.com/citations?user=HyuvK6AAAAAJ&hl=en">Mark J. Margres</a></u>.</font>

<h2>Characterizing Transmission Dynamics of the Tasmanian Devil Facial Tumor Disease, an Infectious Cancer</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/DFTD.jpg" class="inline">
</figure>
<font size="3">Of paramount importance to the conservation of species diversity is an understanding of emerging infectious diseases (EIDs). Frequently driven by novel interactions among previously isolated species, EIDs have been implicated in recent declines or even extinctions in the cases of Dutch elm disease, chytrid fungus in amphibians, and avian malaria an Hawaiian birds. Given the increased prevalence of habitat alteration and severity of global climate change, geographic ranges of many species, including vectors of EIDs are expected to change dramatically. Such range shifts have the potential to facilitate both the spread and establishment of EIDs, as was observed recently with the emergence of Zika virus in Brazil of early 2015. In this study, we aim to characterize the transmission dynamics among lineages of one such EID, the Tasmanian Devil Facial Tumor Disease (DFTD), using a phylodynamic approach. Leveraging thousands of loci captured across the genomes of serially-sampled tumors collected from Tasmanian Devils <i>Sarcophilus harrisii</i>, we will be able to identify tumor strains exhibiting greater virulance. With this information, we then may begin to identify those components of DFTD evolution that led to these increased lineage fitnesses. This work is in collaboration with <u><a href="http://public.wsu.edu/~storfer/">Andrew Storfer (PI)</a></u>, <u><a href="https://www.mccallum-disease-ecology.com/">Hamish McCallum (Co-PI)</a></u>, <u><a href="http://www.utas.edu.au/zoology/research/conservation-biology-and-wildlife-management/effects-of-dftd-on-behaviours-associated-with-transmission-and-on-reproductive-females/menna-jones">Menna Jones (Co-PI)</a></u>, <u><a href="http://hohenlohelab.github.io/">Paul Hohenlohe</a></u>, <u><a href="https://kelleylab.wordpress.com/">Joanna Kelley</a></u>, and <u><a href="https://scholar.google.com/citations?user=HyuvK6AAAAAJ&hl=en">Mark Margres</a></u>.</font>

