---
title: "Ongoing Research"
layout: single
author_profile: true
permalink: /ongoing-research/
excerpt: "I do things <br /> <br /> <br /> <br /> <br /> <br /> <br /> <br />"
header:
  overlay_image: austinhpatton.github.io/Images/PjorOverhead.jpg
  overlay_filter: 0.3
  caption: "<i>Plethodon jordani</i> Dan Mele Photography"
---
<h2>Assessing the role of hybridization in macroevolutionary diversification</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/HybDivers_Image.jpg">
</figure>
<font size="3"> The role of hybridization has had a long contentious role in biology. Thought of as either facilitating and accelerating the lineage diversification process or as constraining it, the majority of research conducted on this topic has been at relatively small taxonomic scales. I am currently working to address this question in extant salamanders using a model of trait-dependent diversification (Hidden-State Speciation and Extinction: HiSSE). We have found that hybridization appears to accelerate speciation rates, resulting in a heightened net-diversification rate relative to non-hybridizing taxa. Thus, our results favor a view of hybridization as a generative process in species diversification. This work is in collaboration with <u><a href="http://lukejharmon.github.io/">Luke Harmon</a></u>, Brendan Epstein, and <u><a href="https://public.wsu.edu/~storfer/eastman/pages/index.html">Jon Eastman</a></u>.</font>

<h2>Exploring the relation between genome size and body size</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/GenomeBodySize_Image.jpg" class="inline">
</figure>
<font size="3">Across both plants and animals, genome size has been found to correlate with body size. This is a consequence of cell size scaling with nuclear volume and physiological constraints associated with the ratio between the two. In salamanders however, this relationship appears to vary among groups. Their massive genome size, paired with their small physical size, has earned them the title 'macroscopic microfauna', referring to their small effective 'biological size.' In this study, we are looking to explore this relationship in salamnders, using a model that allows us to identify uniquely evolutionarily adapted groups and the nature of this relationship in each group respectively. Thus, we question the generality of the conclusions made regarding this relationship across the tree of life. This work is in collaboration with <u><a href="http://uyedaj.github.io/">Josef Uyeda</a></u>.</font>

<h2>Investigating the population and landscape genomics of hybridization</h2>
<figure style="width: 350px" class="wrap align-left">
<img src="/Images/CompLangGenomics_Image.jpg" class="inline">
</figure>
<font size="3">The Southern Appalachians contain the greatest diversity of salamanders in the world. Of the salamanders native to this region, the <i>Plethodontid</i> (terrestrial lungless) salamanders are the most diverse, and many of these have been observed to hybridize in nature. In some cases, hybrizing species pairs hybridize in some locations but not in others. This incomplete reproductive isolation begs the question of how this group came to become so diverse if hybridization is so frequent. In this study, we aim to leverage population and landscape genomic approaches to address this question. Specifically, we aim to identify genomic regions associated with local adaptation and adaptive introgression among three hybridizing species (<i>Plethodon jordani, Plethodon metcalfi, Plethodon teyahalee</i>). To accomplish this, we will use sequence capture to target loci found to be differentially expressed using transcriptomic data. This work is in collaboration with <u><a href="https://kelleylab.wordpress.com/">Joanna Kelley</a></u>.</font>
